HackYourFuture-CPH/FP-annotatetheweb-class10
# #213 Redirect to Register Page

On Login Page, upper right corner (FormLoginRegister component), I added a 'href' to the link, so if user clicks on it, then they will be redirect to Register page. 
**How to test:** 
Run the app, go to '/login'. Click on 'Register' in the upper right corner. See if it takes you to /register page.
